src/transformers/models/auto/tokenization_auto.py

```diff
@@ -185,8 +185,6 @@
         (LongformerConfig, (LongformerTokenizer, LongformerTokenizerFast)),
         (BartConfig, (BartTokenizer, BartTokenizerFast)),
         (LongformerConfig, (LongformerTokenizer, LongformerTokenizerFast)),
-        (RobertaConfig, (BertweetTokenizer, None)),
-        (RobertaConfig, (PhobertTokenizer, None)),
```

Comment:
Those were not in the `values()` of this dict since they are replaced in the next line.

Reply:
Hmm good point.
I feel like in the end this issues comes from the fact that we should have a tokenizer config instead of piggy-backing on the model config. What. do you think @LysandreJik @julien-c ?

Reply:
Yes, we should. This is an issue I'll try to solve next week.